docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -2182,6 +2253,8 @@ even when it is not a valid upgrade from one version to the next.
 **Note:** We still recommend against depending on Daml Script for
 ledger-uploaded packages, even in this case with non-serializable positions.
 
+.. _testing:
```

Comment:
These labels are docs wide, I'd call it something like `upgrades-testing`. Same for the `best_practices`

Reply:
(I believe we also prefer dashes to underscores, other than ofc the one at the start. I'd check some others to see if thats true)

Reply:
I've updated these two, along with a bunch of other ones in the doc